st2common/st2common/models/api/keyvalue.py

```diff
@@ -57,8 +76,39 @@ class KeyValuePairAPI(BaseAPI):
         'additionalProperties': False
     }
 
+    @staticmethod
+    def _setup_crypto():
```

Comment:
I would imagine encryption and decryption for kvp being handled in the persistence or the db modules. This way its not an API only feature and it would be possible to get decrypted keys even if not using the API i.e. from services do not use the API layer.